<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshReconstruction</class>
 <widget class="QMainWindow" name="MeshReconstruction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraspStudio - GraspPlanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButtonLoadPoints">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Points</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonLoadObject">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>0</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Object</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelInfo">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>670</y>
         <width>271</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>Mesh Info</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBoxVisualization">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>281</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string>Visualization</string>
       </property>
       <widget class="QCheckBox" name="checkBoxColModel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Collision Model</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxObject">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Object</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxPoints">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Points</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxReconstruction">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Reconstruction</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButtonReconstruct">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Reconstruct Mesh</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
